root/etc/s6-overlay/s6-rc.d/init-openssh-server-config/run

```diff
@@ -132,5 +132,13 @@ chmod 700 \
     /config/.ssh
 chmod 600 \
     /config/.ssh/authorized_keys
-chown root:root \
+
+# ssh_host_keys permissions
+lsiown -R root:root \
+    /config/ssh_host_keys
```

Comment:
Apologies if I'm way off - why not leave it as "${USER_NAME}":"${USER_NAME}" and possibly 700/750? In cases where /config is an external volume mounted into the container with stricter permissions, this would widen the perms.

Reply:
Because a persistent complaint about this image is the ability for a connected user to modify the sshd_config, which would take effect on a container restart. You can't just chown sshd_config to root because an unprivileged user can still overwrite it with their own file.

Another option might be chown the folder and sshd_config to root but leave the host keys owned by the running user. This has the advantage of not having to change the permissions on the host keys but the permissions on the folder and config would still need to be world readable.

Reply:
Got it, thanks for clarifying. Would it work to chown the sshd_config/folder to root:${USER_NAME} with 640/750 permissions?

Reply:
Possibly, I'll test and see